test/api/candles.js

```diff
@@ -7,11 +7,11 @@ describe('Candles API', function() {
 
     /*Define functions for use within tests*/
     function getCandles(e, d, done) {
-        node.get('/api/candles/getCandles?e=' + e + '&d=' + d, done);
+        node.get('/api/exchanges/getCandles?e=' + e + '&d=' + d, done);
```

Comment:
I think we should rename this test file (and context) to: `test/api/exchanges.js`.